code/FulpstationCode/fulpspaceruins.dm

```diff
@@ -151,6 +151,29 @@ There is no point for a testament, but maybe a confession is appropriate. I've n
 	new/obj/structure/fluff/empty_sleeper/syndicate(get_turf(src))
 	return ..()
 
+/obj/effect/mob_spawn/human/old_guard_captain
+	name = "Old Guard Captain"
+	roundstart = FALSE
+	death = FALSE
+	random = TRUE
+	mob_name = "Old Guard Captain"
+	icon = 'icons/obj/machines/sleeper.dmi'
+	icon_state = "sleeper"
+
+	short_desc = "You are a Captain from a forgotten era."
+	flavour_text = "Once the pride of the navy, your ship has fallen into disrepair from abandonement and years of disuse. You've just woken up, and darkness embraces you..."
+	important_info = "You are not an antagonist! Do not harm other beings!"
```

Comment:
```suggestion
	important_info = "You are not an antagonist! Do not harm other sapient beings!"
```

Reply:
alternatively, since you're already referring to the meta term of "antagonist", you can just outright say to not harm other player-controlled characters.